common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -57,10 +57,10 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
-  val AdFreeTrialSwitch = Switch(
+  val AdFreeEmergencyShutdown = Switch(
     Commercial,
-    "ad-free-subscription-trial",
-    "Master switch for trialling ad-free subscription",
+    "ad-free-emergency-shutdown",
+    "When ON, we kill the ad-free service immediately.",
     owners = Seq(Owner.withGithub("JustinPinner")),
     safeState = Off,
     sellByDate = never,
```

Comment:
This works in the opposite way to all the other feature switches.  Would it be better to call it something like 'AdFreeExperience' or just 'AdFree' and have it switched on by default?

Reply:
That's a fair comment, and I could have made it clearer in code that this was intentional. It's because the default state for ad-free is going to be 'on' and shouldn't be switchable, but we want to retain the ability to to shut it down if we spot something weird happening during the next couple of weeks live testing. By the time the comms go out and it becomes a permanent feature, this switch will be removed.

Reply:
If it's just a temporary switch could you give it an expiry date?

Reply:
Yes. I should've thought of that! 🥇